lgi/override/Gtk.lua

```diff
@@ -219,6 +219,9 @@ if Gtk.Container then
    end
 end
 
+-------------------------------- Gtk.Box overrides.
+Gtk.Box._container_add = Gtk.Box.append
```

Comment:
Good idea. This method doesn't exist in GTK3. I'll look into moving this down the bottom the file inside a "GTK4 overrides" block, right next to the old GTK2 check.

Reply:
as of the latest commit:
<details><summary> Gtk override </summary>
<p>

```
-------------------------------- Gtk-4 overrides

if Gtk._version == '4.0' then

   --- CONTAINER CLASSES
   --- GTK4 Has removed the Container abstract class.
   --- This means Boxes, Grids and other layout widgets 
   --- Must now add children using their own specific methods.

   --- Gtk.Box overrides

   --- widgets on a the array part of the constructor of a Box
   --- will be `append()`-ed
   --- in order of appearance.
   Gtk.Box._container_add = Gtk.Box.append

end
```

</p>
</details> 

This will prevent Gtk versions != 4.0 from attempting `Gtk.Box.append()`.